src/modules/bookmark/utils.js

```diff
@@ -26,17 +25,25 @@ export const validateBookmarks = (data) => {
 export const getIndexOfBookmark = (bookmarks, { address, token = tokenMap.LSK.key }) =>
   bookmarks[token].findIndex((bookmark) => bookmark.address === address);
 
+export const getIndexOfLabel = (bookmarks, { label, token = tokenMap.LSK.key }) =>
+  bookmarks[token].findIndex((item) => item.title === label);
+
 /**
  *  Checks the label and returns feedback
  *
+ * @param {String} token - LSK or any other token
  * @param {String} value - The label string to check
+ * @param {Object} bookmarks - List of bookmarks from Redux store
  * @param {Function} t - i18n function
  * @returns {String} - Feedback string. Empty string if the label is valid
  */
-export const validateBookmarkLabel = (value = '', t) => {
+export const validateBookmarkLabel = (token, value = '', bookmarks, t) => {
```

Comment:
Why is token part of bookmark validation?

Reply:
Bookmarks are stored by token so it makes sense to pass in the active token

Reply:
Refer my above comment.. we shouldn't store bookmark by token.. they are generic?

Reply:
To be done in a different PR